style.css

```diff
@@ -621,4 +606,70 @@ header {
   .contact {
     margin-right: 0;
   }
+
+  #mobile-buttons {
+    display: none;
+  }
+}
+
+/*** Closing javascript button ***/
+.mobile-menu {
+  position: absolute;
+  background: #6070ffee;
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(8px);
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
+  display: none;
+}
```

Comment:
- [ ] Kindly add a blur effect for the content behind the menu like in the Figma design.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/179723679-03ac1391-95ab-43fa-a683-b7817929ad1e.png) | ![image](https://user-images.githubusercontent.com/53402600/179723746-c49ccf31-fb0f-4a26-9a3e-45ab54684c92.png) |